runtime/pendulum/src/lib.rs

```diff
@@ -908,6 +912,23 @@ impl dia_oracle::Config for Runtime {
 	type WeightInfo = dia_oracle::weights::DiaWeightInfo<Runtime>;
 }
 
+parameter_types! {
+	pub const FarmingKeeperPalletId: PalletId = PalletId(*b"fo/fmkpr");
+	pub const FarmingRewardIssuerPalletId: PalletId = PalletId(*b"fo/fmrir");
```

Comment:
Nit: The `fo` indicates foucoco, on Amplitude we use `am` so let's use `pe` here.
```suggestion
	pub const FarmingKeeperPalletId: PalletId = PalletId(*b"pe/fmkpr");
	pub const FarmingRewardIssuerPalletId: PalletId = PalletId(*b"pe/fmrir");
```

Reply:
Oh I didn't notice. Committing.